src/Parsing/Impl/Ast/PythonNameBinder.cs

```diff
@@ -85,34 +90,35 @@ internal class ParameterBinder : PythonWalkerNonRecursive {
                 }
             }
         }
+
         public override bool Walk(TupleExpression node) => true;
     }
 
     class DeleteBinder : PythonWalkerNonRecursive {
         private readonly PythonNameBinder _binder;
+
         public DeleteBinder(PythonNameBinder binder) {
             _binder = binder;
         }
+
         public override bool Walk(NameExpression node) {
             _binder.DefineDeleted(node.Name);
             return false;
         }
     }
 
-    class PythonNameBinder : PythonWalker {
-        private ScopeStatement _currentScope;
+    public class PythonNameBinder : PythonWalker {
```

Comment:
`public` got added here, but before it was the default, which I think is "internal", so maybe use that if you'd like to be explicit.

Reply:
ok